example_project/example_project/settings.py

```diff
@@ -53,34 +53,37 @@
 
 WSGI_APPLICATION = 'example_project.wsgi.application'
 
+# Internationalization
+# https://docs.djangoproject.com/en/1.6/topics/i18n/
 
-# Database
-# https://docs.djangoproject.com/en/1.6/ref/settings/#databases
+DEBUG=True
```

Comment:
Our code guide lines and django default settings (see below) propose a whitespace before and after the assign operator. Please use same style everywhere.